docs/patterns/components/ClickOutsideListener/index.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import { ClickOutsideListener, Code, Button } from '@deque/cauldron-react/';
+import PropDocs from '../../../Demo/PropDocs';
 
 const Demo = () => (
   <div>
     <h1>Click Outside Listener</h1>
+    <h2>Component Description</h2>
+    <p>
+      If a user clicks outside of this component on the page, the{' '}
+      <code>onClickOutside</code> method will be called.
+    </p>
```

Comment:
Tricky one. Maybe `a component wrapper that calls a method when clicking outside the wrapped component`?